mne_bids_pipeline/tests/configs/config_funloc.py

```diff
@@ -0,0 +1,36 @@
+"""Funloc data."""
+
+from pathlib import Path
+
+data_root = Path("~/mne_data").expanduser().resolve()
+bids_root = data_root / "MNE-funloc-data"
+deriv_root = data_root / "derivatives" / "mne-bids-pipeline" / "MNE-funloc-data"
+subjects_dir = bids_root / "derivatives" / "freesurfer" / "subjects"
+task = "funloc"
+ch_types = ["meg", "eeg"]
+data_type = "meg"
+
+# filter
+l_freq = None
+h_freq = 50.0
+# maxfilter
+use_maxwell_filter: bool = True
+mf_st_duration = 60.0
+# SSP
+n_proj_eog = dict(n_mag=1, n_grad=1, n_eeg=2)
+n_proj_ecg = dict(n_mag=1, n_grad=1, n_eeg=0)
+
+# Epochs
+epochs_tmin = -0.2
+epochs_tmax = 0.5
+epochs_t_adjust = -4.0e-3  # TODO is there a way to do this in MBP?
```

Comment:
@larsoner does MBP have a way of doing what `t_adjust` does in mnefun?

Reply:
No, this should be fixed/adjusted when bidsifying the dataset I think (the timings, event types, etc. in there should be as accurate as possible and not require fixing)